5.5/run-mysqld-slave.sh

```diff
@@ -6,6 +6,7 @@ source ${HOME}/common.sh
 set -eu
 
 export MYSQL_RUNNING_AS_SLAVE=1
+export MYSQL_MASTER_ADDR=${MYSQL_MASTER_ADDR:-mysql-master}
```

Comment:
@bparees +1 we should use the service name instead of hard coding 'mysql-master' here.